x-pack/plugins/cases/server/client/typedoc_interfaces.ts

```diff
@@ -55,3 +56,5 @@ export interface ISubCaseResponse extends SubCaseResponse {}
 export interface ISubCasesResponse extends SubCasesResponse {}
 
 export interface ICaseUserActionsResponse extends CaseUserActionsResponse {}
+
+export interface IAlertResponse extends AlertResponse {}
```

Comment:
I don't we need this. The `AlertResponse` is pretty small so we can just display it as it is.

Reply:
Oh I see! It is mostly for intersections and unions, right?